modules/distribution/src/main/config/fluo-env.sh

```diff
@@ -37,7 +37,7 @@ export FLUO_TMP="${FLUO_TMP:-/tmp}"
 ## Fluo log4j configuration
 export FLUO_LOG4J_CONFIG="${FLUO_LOG4J_CONFIG:-${conf}/log4j.properties}"
 ## Java options along with FLUO_JAVA_OPTS for Fluo command
-JAVA_OPTS=("${FLUO_JAVA_OPTS[@]}" "-Dlog4j.configuration=file:${FLUO_LOG4J_CONFIG}")
+JAVA_OPTS=("${FLUO_JAVA_OPTS[@]}")
```

Comment:
Do you remember why we were explicitly setting the config instead of picking it up from the class path? We can still set it explicitly... but the property name might be slightly different for log4j2. But, I'm not sure why we previously had it, so it may not be necessary.

Reply:
I have no idea why we were setting it.  When this property was set to a non-existent file it caused log4j2 to blow up with some weird error message with nulls in it.  Took me a bit to figure that out.  The error message was not helpful.  I need to remove the export a few lines up.